lib/doorkeeper/config/option.rb

```diff
@@ -53,7 +53,7 @@ def option(name, options = {})
             value = if attribute_builder
                       attribute_builder.new(&block).build
                     else
-                      block || args.first
+                      block || (options[:array] == true ? args : args.first)
```

Comment:
We previously didn't have a way to use the `option` DSL to define options that take multiple arguments and convert them to a list/array. This seemed like a reasonable thing to add?

Reply:
If we choose to add this, I don't mind if it's as `options[:array]` or `options[:list]` or something.